Unit test failure

Unit tests are failing for me (3/7) in main branch. Django version `4.2.2`. Python version `3.10.3`

```
django-admin test --settings=tests.settings --pythonpath=.

Found 7 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
F...FF.

----------------------------------------------------------------------
Ran 7 tests in 0.044s

FAILED (failures=3)
```

Regression in b1290953175916780e0116d6501624f52678c04a 

```
======================================================================
FAIL: test_list_empty (tests.tests.BasicTests.test_list_empty)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/carlton/Projects/Django/neapolitan/tests/tests.py", line 56, in test_list_empty
    self.assertContains(
  File "/Users/carlton/Envs/neapolitan/lib/python3.11/site-packages/django/test/testcases.py", line 660, in assertContains
    self.assertTrue(
AssertionError: False is not true : Couldn't find '<a href="/bookmark/new/">Add a new bookmark</a>' in response
```

Need to look at why that's not the case.

--- a/tests/tests.py
+++ b/tests/tests.py
@@ -44,18 +44,14 @@ class BasicTests(TestCase):
         self.assertContains(response, self.homepage.title)
         self.assertContains(response, self.github.title)
         self.assertContains(response, self.fosstodon.title)
-        self.assertContains(
-            response, '<a href="/bookmark/new/">Add a new bookmark</a>', html=True
-        )
+        self.assertContains(response, ">Add a new bookmark</a>")
 
     def test_list_empty(self):
         Bookmark.objects.all().delete()
         response = self.client.get("/bookmark/")
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "There are no bookmarks. Create one now?")
-        self.assertContains(
-            response, '<a href="/bookmark/new/">Add a new bookmark</a>', html=True
-        )
+        self.assertContains(response, ">Add a new bookmark</a>")
 
     def test_detail(self):
         response = self.client.get(f"/bookmark/{self.homepage.pk}/")
@@ -69,7 +65,7 @@ class BasicTests(TestCase):
         # Load the form.
         response = self.client.get(create_url)
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, '<form method="POST" action="/bookmark/new/">')
+        self.assertContains(response, 'action="/bookmark/new/"')
 
         # Submit the form.
         response = self.client.post(

